src/Shouldly/Shouldly.csproj

```diff
@@ -14,7 +15,7 @@
     <PackageProjectUrl>https://shouldly.org</PackageProjectUrl>
     <PackageLicenseExpression>BSD-2-Clause</PackageLicenseExpression>
     <RepositoryType>git</RepositoryType>
-    <RepositoryUrl>https://github.com/shouldly/shouldly.git</RepositoryUrl>
+    <RepositoryUrl></RepositoryUrl> <!-- Is set by Microsoft.SourceLink.GitHub -->
```

Comment:
```suggestion
```

Reply:
The only benefit is to enable SourceLink in forks automatically. Otherwise, it would be out of sync with the sources in the origin repository.